recipes/openblas/all/conanfile.py

```diff
@@ -19,134 +23,219 @@ class OpenblasConan(ConanFile):
         "fPIC": [True, False],
         "build_lapack": [True, False],
         "use_thread": [True, False],
+        "use_openmp": [True, False],
         "dynamic_arch": [True, False],
+        "with_avx512": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "build_lapack": False,
         "use_thread": True,
+        "use_openmp": False,
         "dynamic_arch": False,
+        "with_avx512": False,
     }
-    generators = "cmake"
+    package_type = "library"
     short_paths = True
 
+    def _fortran_runtime(self, fortran_id):
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            if fortran_id == "GNU":
+                if self.settings.compiler == "gcc":
+                    # Compiler vs. gfortran runtime ver.: 5,6: 3, 7: 4, >=8: 5
+                    if Version(self.settings.compiler.version).major >= "5":
+                        return "gfortran"
+                if self.settings.compiler == "clang":
+                    if Version(self.settings.compiler.version).major > "8":
+                        return "gfortran"  # Runtime version gfortran5
+
+        self.output.warning(
+            f"Unable to select runtime for Fortran {fortran_id} "
+            f"and C++ {self.settings.compiler} {self.settings.compiler.version}")
+        return None
+
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _openmp_runtime(self):
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            if self.settings.compiler == "gcc":
+                if Version(self.settings.compiler.version).major > "7":
+                    return "gomp"
+            if self.settings.compiler == "clang":
+                if Version(self.settings.compiler.version).major > "8":
+                    return "omp"
+        return None
 
     @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    def _fortran_compiler(self):
+        comp_exe = self.conf.get("tools.build:compiler_executables")
+        if comp_exe and 'fortran' in comp_exe:
+            return comp_exe["fortran"]
+        return environ.get('FC')
```

Comment:
```suggestion
        return None
```
I would keep only conan conf support for safety. So when people report some error, they need to share their profile too. Plus, env vars are easily manipulated and may cause a mess, and that's the influence of adopting conf in Conan 2.x

Reply:
Added suggestion in commit